project/Build.scala

```diff
@@ -4,18 +4,22 @@ import Keys._
 object MediachainBuild extends Build {
   updateOptions := updateOptions.value.withCachedResolution(true)
 
+  val specs2Version = "3.7.3"
+  val scalaCheckVersion = "1.13.1"
+
   override lazy val settings = super.settings ++ Seq(
     organization := "io.mediachain",
     version := "0.0.1",
     scalaVersion := "2.11.7",
     scalacOptions ++= Seq("-Xlint", "-deprecation", "-Xfatal-warnings", "-feature"),
     libraryDependencies ++= Seq(
       "org.typelevel" %% "cats" % "0.4.1",
-      "org.json4s" %% "json4s-jackson" % "3.2.11",
-      "org.specs2" %% "specs2-core" % "3.7" % "test",
-      "org.specs2" %% "specs2-junit" % "3.7" % "test",
-      "org.specs2" %% "specs2-matcher-extra" % "3.7" % "test",
-      "org.scalacheck" %% "scalacheck" % "1.13.0" % "test"
+      "org.json4s" %% "json4s-jackson" % "3.3.0",
```

Comment:
dumb question: is there an easy way to dump out cbor as something human-readable for debugging?